website_sale_brand_catalogs/views/pages/catalogs.xml

```diff
@@ -0,0 +1,321 @@
+                                                        required=""
+                                                        data-fill-with="email"
+                                                    />
+                                                </div>
+                                            </div>
+                                            <div data-type="char" data-name="Field">
+                                                <div class="form-group form-field o_website_form_required_custom">
+                                                    <input
+                                                        id="contact3"
+                                                        type="tel"
+                                                        class="form-control s_website_form_input"
+                                                        name="phone"
+                                                        placeholder="Phone*"
+                                                        required=""
+                                                        data-fill-with="phone"
+                                                    />
+                                                </div>
+                                            </div>
+                                            <div
+                                                class="form-group col-12 s_website_form_submit"
+                                                data-name="Submit Button"
+                                            >
+                                                <div class="s_website_form_label" />
+                                                <a
+                                                    data-attachment-id=""
+                                                    data-pdf-mode=""
+                                                    role="button"
+                                                    class="btn btn-primary btn-lg btn-block text-white download_attachment"
+                                                >Download</a>
+                                                <span id="s_website_form_result" />
+                                            </div>
+                                            <div class="form-group form-field hidden">
+                                                <t t-set="section_name">Catalog download:</t>
+                                                <t t-set="catalog_download" t-value="section_name" />
+                                                <input
+                                                    class="form-control o_website_form_input"
+                                                    name="name"
+                                                    placeholder=""
+                                                    t-att-value="'%s %s (%s)' % (catalog_download, catalog.attachment_id.name, catalog.name)"
```

Comment:
This line is too long, please split:

```suggestion
                                                    t-att-value="'%s %s (%s)' % (
                                                        catalog_download,
                                                        catalog.attachment_id.name,
                                                        catalog.name,
                                                    )"
```

Reply:
Already commented it, but unresolving this thread because here is the expected form

Reply:
Just missing the closing parenthesis part